src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void printList(Task[] list) {
+        System.out.println("     Here are the tasks in your list:");
+        for(int i = 0; i < list.length; i ++) {
+            if(list[i] == null) {
+                break;
+            }
+            System.out.println("     " + (i + 1) + "[" + list[i].getStatusIcon() + "] " + list[i].description);
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Task[] list = new Task[100];
+        String greeting = "    ____________________________________________________________\n"
+                + "     Hello! I'm Duke\n"
+                + "     It seems like you are needing some help.\n"
+                + "    ____________________________________________________________\n";
+        int listCount = 0;
+        System.out.println(greeting);
+        while(true) {
+            Scanner command = new Scanner(System.in);
+            String echoCommand = command.nextLine();
+            String[] words = echoCommand.split(" ");
+            System.out.println("    ____________________________________________________________");
+            if(echoCommand.equals("bye")) {
+                System.out.println("     Bye. Hope to see you again soon!");
+                System.out.println("    ____________________________________________________________\n");
+                break;
+            }
+            else if(echoCommand.equals("list")) {
+                printList(list);
+            }
+            else if(words.length == 2 && words[0].equals("done") && (int)Double.parseDouble(words[1])
```

Comment:
This else-if statement is really complex, it took me some time before I could understand your code 😅 Maybe you could shorten it to individual boolean variables or more descriptive conditions, or even minimally commenting what this else-if statement does?

Reference: https://nus-cs2113-ay1920s2.github.io/website/book/codeQuality/maximiseReadability/basic/avoidComplicatedExpressions/